metadata/location/utah_address_points

```diff
@@ -0,0 +1,84 @@
+# Title
+
+Utah Address Points
+
+## ID
+
+4373453d-9e20-44da-9c6e-1f0b1ea653af
+
+## Brief Summary
+
+Contains US Postal Service address points for all twenty-nine counties in Utah.
+
+## Summary
+
+An address point represents a geographic location that has been assigned a US Postal Service (USPS) address by the local address authority (i.e., county or municipality) but does not necessarily receive mail.
+
+## Description
+
+### What is the dataset
+
+This dataset contains vector point data for over one million addresses in the State of Utah and includes information such as the full street address and the type of unit the point represents. This dataset is a mapping-grade framework layer that receives regular updates. Maintaining this dataset is an ongoing effort between UGRC and county-level data stewards.
+
+### What is the purpose of the dataset
+
+Address data are used for a variety of purposes, including [geocoding](api.mapserv.utah.gov), city planning, and [redistricting](https://redistricting.utah.gov/wp-content/uploads/2021/07/Redistricting-Packet-1-compressed.pdf). In addition, UGRC actively collaborates with the US Postal Service and Google Maps to improve mail and package delivery, navigation apps, and emergency services.
+
+This dataset has also been used by the [Utah Broadband Center](https://business.utah.gov/broadband/) and UGRC to determine areas with limited internet access in an effort to improve services in underserved communities. You can learn more about the Utah Broadband Project on our [blog](https://gis.utah.gov/blog/2016-02-01-utah-broadband-outreach-center-releases-revamped-availability-map/).
+
+### What does the dataset represent
+
+Points are generally placed on the structure of the address, but may also be found on the driveway or otherwise within the parcel of land. In cases where the county has provided the specific location of the address point for a given address, it is denoted in the "PtLocation" field. The address points in this dataset also include information such as:
```

Comment:
```suggestion
Points are generally placed on the structure associated with the address, but may also be found on the driveway or otherwise within the parcel of land. In cases where the county has provided the specific location of the address point for a given address, it is denoted in the "PtLocation" field. The address points also include information such as:
```